gammapy/utils/pbar.py

```diff
@@ -0,0 +1,26 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+"""Utilities for progress bar display"""
+import logging
+from contextlib import contextmanager
+
+log = logging.getLogger(__name__)
+
+try:
+    from tqdm import tqdm
```

Comment:
I think importing from `tqdm.auto` would be still good. You mentioned it does not work for you...does it fail, or do you just get the "normal" console progress bar?

Reply:
@adonath done. I realized I was using a very old jupyter lab version (~1.0). After updating to jupyterlab 3.0, the bar is correctly rendered